src/tokens/polygonTokens.json

```diff
@@ -20519,4 +20519,22 @@
             }
         }
     }
```

Comment:
Missing a comma "," after the previous entry which is breaking the build process. Also check indentation. you can check for errors running "npm run lint".

Reply:
Hi, updated the error with new pull request, please check and do the needful. Thank you.